Anytype/Sources/PresentationLayer/Templates/Views/Selection/EditableView.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+struct EditableView<Content: View>: View {
+    var content: Content
+    var onEditingTap: () -> Void
+    var canBeEdited: Bool
+    @Binding var isEditing: Bool
+    
+    var body: some View {
+        ZStack(alignment: .topTrailing) {
+            content
+                .padding(.trailing, 8)
+                .padding(.top, 8)
+            if canBeEdited && isEditing {
+                dotImageButton
+            }
+        }
+    }
+    
+    var dotImageButton: some View {
+        Button {
+            onEditingTap()
+        } label: {
+            dotImage
+        }
+    }
+    
+    var dotImage: some View {
+        ZStack {
+            VisualEffectView(effect: UIBlurEffect(style: .light))
```

Comment:
Doesn't possible to use `.background(.thinMaterial)` here?

Reply:
Not the same effect unfortunately. I tried it.